Client/Frontend/Brave Rewards/Common/LegacyWalletTransferStatusButton.swift

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Shared
+import BraveUI
+
+class LegacyWalletTransferStatusButton: UIControl, Themeable {
+    
+    let dismissButton = Button(type: .system).then {
+        $0.setImage(UIImage(imageLiteralResourceName: "close-medium").template, for: .normal)
+        $0.tintColor = .white
+        $0.hitTestSlop = UIEdgeInsets(equalInset: -10)
+        $0.setContentHuggingPriority(.required, for: .horizontal)
+    }
+    
+    let titleLabel = UILabel().then {
+        $0.font = .systemFont(ofSize: 18, weight: .medium)
+        $0.numberOfLines = 0
+        $0.appearanceTextColor = .white
+        $0.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        isAccessibilityElement = true
+        accessibilityTraits.insert(.button)
+        
+        layer.cornerRadius = 8
+        layer.cornerCurve = .continuous
+        
+        backgroundColor = UIColor(rgb: 0x339AF0)
```

Comment:
is this color undefined in our color constants?

Reply:
Yeah I dunno James just used a new blue colour we've never used before in these designs lol